builder/alicloud/ecs/builder_acc_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"testing"
 
 	"github.com/aliyun/alibaba-cloud-sdk-go/services/ecs"
-	builderT "github.com/hashicorp/packer/helper/builder/testing"
+	builderT "github.com/hashicorp/packer/packer-plugin-sdk/builder/testing"
```

Comment:
Once broken out into a separate ropo, what do you think about renaming this to `acctest` or creating a new pkg that allows for the acceptance testing of builds including provisioners and post-processors?

Reply:
yeah I like that idea; changing the name to acctest would make it a lot clearer architecturally what's going on. I'll do that in a final PR once I've finished cleaning up the package and its usage.